src/population/visualization/family_tree_tracker.py

```diff
@@ -0,0 +1,201 @@
+from datetime import datetime
+import json
+import networkx as nx
+import os
+import numpy as np
+from loguru import logger
+from population.visualization.visualization import create_and_save_figure, make_dir_if_not_exists
+from population.visualization.graphing_model import get_neural_net_positions
+from population.visualization.gene_data_processing import convert_genes_to_numerical, get_pca_positions, get_pca_colors
+
+class FamilyTreeTracker:
```

Comment:
I am eventually going to move these files in the visualization directory somewhere else but I don't know where yet.

Reply:
For now they should at least be in the exastar submodule since most of this stuff is hardcoded for those genomes.